pkg/config/token_signing_config.go

```diff
@@ -25,33 +26,79 @@ import (
 type TokenSigningConfig struct {
 	// Keys determines the key manager configuration for this token signing
 	// configuration.
-	Keys keys.Config `env:",prefix=TOKEN_"`
+	Keys keys.Config `env:", prefix=TOKEN_"`
 
-	TokenSigningKeys   []string `env:"TOKEN_SIGNING_KEY, required"`
+	// TokenSigningKeys is the parent token signing key (not the actual signing
+	// version). It is an array for backwards-compatibility, but in practice it
+	// should only have one element.
+	//
+	// Previously it was a list of all possible signing key versions, but those
+	// have moved into the database.
+	//
+	// TODO(sethvargo): Convert to string in 0.22+.
```

Comment:
we'll have to tackle the terraform changes as well...

Reply:
They aren't actually that bad. This is the only change.